src/lp/empty_column.jl

```diff
@@ -1,10 +1,71 @@
+@doc raw"""
+    RemoveEmptyColumn <: AbstractRule
+
+Eliminate a single column if it is empty, i.e. all coefficients are zero.
+
+## Presolve
+
+If column $j$ is empty, variable $x_{j}$ is fixed to $x_{j}^{*}$ as follows
+
+| $c_{j}$ |        $l_{j}$ |        $u_{j}$ | $x^{*}_{j}$ |
```

Comment:
This is all assuming minimization, correct?

Reply:
Correct. I'll mention it in the docs.